regression/cases/fluent_elasticsearch/lading/lading.yaml

```diff
@@ -2,7 +2,7 @@ generator:
   - tcp:
       seed: [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53,
         59, 61, 67, 71, 73, 79, 83, 89, 97, 101, 103, 107, 109, 113, 127, 131]
-      addr: "localhost:8282"
+      addr: "0.0.0.0:8282"
```

Comment:
I'm somewhat surprised that `0.0.0.0` works here; lading is connecting to this and I didn't know Linux supported connections to that address. (I'd have expected `127.0.0.1` here.) No change needed, just curious.

Reply:
It is possible that this specific one wasn't needed, I just took a "big hammer" approach and made everything `0.0.0.0`

Reply:
I'm also a bit surprised by this. Did you find this change necessary? I could see it causing confusion in the future. I'd also expect `localhost` here.

Reply:
I'll try with `localhost` in a followup PR to see.

Reply:
The hostname `localhost` is broken inside target containers (never investigated; it's a longstanding bug that'd be really nice to have fixed.)